### PR TITLE
Upgrading to use both existing API and new QUnit hooks API.

### PR DESCRIPTION
**Pull request by scalvert**
Upgrading addon to use both global sandbox setup and the new QUnit hooks API

### CODE REVIEW

**Comment by scalvert** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -55,6 +62,26 @@ test('very important test happening here', function(assert) {
...
+### QUnit `hooks` API
+
+To use, import the setup method from within your test file and execute it.
+
+```js
+import setupSinonSandbox from 'ember-sinon-sandbox/test-support';
+
+...
+
+module('my module', {
```
Comment (rwjblue) [inline README.md]: should be `module('my module', function(hooks) {`

**Comment by rwjblue** [comment]
👍 other than that one typo

File: README.md (hunk 1 above)
Comment (scalvert) [reply README.md]: Sorted!

**Pull request merged**